subsys/bluetooth/controller/ll_sw/openisa/README

```diff
@@ -0,0 +1,23 @@
+BLE support on VEGAboard HW  Rev 1.0
+
+Currently the following sample applications are supported by the BLE SW LL
+and BLE host stack running on RI5CY core of RV32M1 SoC:
+
+- BEACON
+- CENTRAL
+- CENTRAL_HR
+- EDDYSTONE
+- HCI_UART
+- IBEACON
+- PERIPHERAL_CSC (Cycling Speed Cadence)
+- PERIPHERAL_DIS (Device Information Service)
+- PERIPHERAL_ESP (Environmental Sensing Service)
+- PERIPHERAL_HR (Heart Rate)
+- PERIPHERAL_HT (Health Thermometer)
+- SCAN_ADV
+
+Limitations:
+- no 2 Mbps, 512/256 Kbps PHY
+- no controller-based cryptographic services
```

Comment:
@alexandru-porosanu-nxp It seems it is not only private address resolution that is not supported, but also LL packet encryption? (if so maybe it would be worth mentioning it explicitly as the 1st one is a relatively advanced feature while the 2nd one is quite basic)

Reply:
Nope, neither RPA nor LL packet encryption are supported for now. I will explicitly state this.